components/ILIAS/Init/classes/class.ilInitialisation.php

```diff
@@ -688,6 +688,27 @@ protected static function setSessionCookieParams(): void
         }
     }
 
+    protected static function initMail(\ILIAS\DI\Container $c): void
```

Comment:
Please don't reintroduce this method, put new registrations to `\ILIAS\Mail\Service\MimeMailService`.